Robot.java_03_22_17

```diff
@@ -1,518 +1 @@
-		System.out.println("\t\t\tright encoder distance	" + rightEncoder.getDistance());
-		System.out.print("left rate: " + leftEncoder.getRate());
-		System.out.println("\t\t\tleft encoder distance	" + leftEncoder.getDistance());
-		//autoDriveStraight(1.5);		
-	if (leftEncoder.getDistance() < feetToEncUnits(5) && rightEncoder.getDistance() < feetToEncUnits(5)) {
-			//rotations: X * 2400 / 360 
-			//feet: 
-			left.set(.3);
-			right.set
-			\\System.out.println("        !        ");
-			
-		}
-		else {
-			System.out.println("    it went 5 feet    ");
-			left.set(0);
-			right.set(0);
-		}  */
-	/*	if (myTimer.get() < (3)) {
-			//rotations: X * 2400 / 360 
-			//feet: 
-			left.set(.1);
-			right.set(.1);
-			System.out.println("        !        ");
-			
-		}
-		else {
-			System.out.println("    3 seconds   ");
-			left.set(0);
-			right.set(0);
-		} 
-		*/
-		
-		//middle lane 
-		System.out.println(state);
-		if (state.equals("middleforward1")) {//midlane
-		//	moveForward(9.00);
-			moveForwardSlow(5);
-			if (finishedMoving(5)) 
-				state = "stop";
-		}
-		
-		//right lane 
-		else if (state.equals("rightforward1")) {
-			moveForward(9.5167);
-			//10.83 feet = 129.96 inches
-			//9.5167 feet = 114.3 inches
-			//have to calculate again with the length of the robot and the distance from the center
-			//from left and right positions not the middle
-			if (finishedMoving(9.5167))
-				state = "rightturn1";
-		}
-		else if (state.equals("rightturn1")) {
-			//this is a right turn
-			changeAngle(10); 
-			if (finishedTurning(10))
-				state = "rightforward2";
-		}
-		else if (state.equals("rightforward2")) {
-			moveForwardSlow(.5);
-			if (finishedMoving(.5))
-				state = "stop";
-		}
-		
-		//leftlane
-		else if (state.equals("leftforward1")) {
-			moveForward(9.5167);
-			if (finishedMoving(9.5167))
-				state = "leftturn1";
-		}
-		else if (state.equals("leftturn1")){//c'est une tournee gauche
-			changeAngle(-120);
-			if (finishedTurning(-120))
-				state = "leftforward2";
-		}
-		else if (state.equals("leftforward2")) {
-			moveForwardSlow(.5);
-			if (finishedMoving(.5));
-				state = "stop";
-		}
-		
-		else if (state.equals("stop")){
-			left.set(0);
-			right.set(0);
-		}
-		else {
-			left.set(0);
-			right.set(0);
-		}
-	}
-	
-	
-	
-	
-	@Override
-	public void teleopInit() {
-		// This makes sure that the autonomous stops running when
-		// teleop starts running. If you want the autonomous to
-		// continue until interrupted by another command, remove
-		// this line or comment it out.
-		System.out.println("Teleop!");
-		
-/*		if (cameraMode.equalsIgnoreCase("BACK")) {
-	//		if (true){
-			cam1 = CameraServer.getInstance().startAutomaticCapture(1);
-		}
-		
-		else if (cameraMode.equalsIgnoreCase("FRONT")) {
-			cam0 = CameraServer.getInstance().startAutomaticCapture(0);
-		}
-*/			
-	
-
-		if (autonomousCommand != null)
-			autonomousCommand.cancel();
-		
-		left.disable();
-		right.disable();
```

Comment:
In reality, it's probably fine right now because the PID controller gets re-created in autoInit. You'll have to test it, going back and forth between init and telep and make sure everything still works. Or, you can just be paranoid and reset the code every match.